### PR TITLE
Minio Chart

### PR DESCRIPTION
**Pull request by hunter**
A basic Minio chart based off the [quickstart guide](https://docs.minio.io/docs/minio-docker-quickstart-guide)

* A single export backed by a configurable PV
* Default Access Key and Secret that can be overridden

### CODE REVIEW

**Comment by viglesiasce** [comment]
@k8s-bot ok to test

File: stable/minio/README.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm install --name my-release stable/minio
+```
+
+The command deploys Minio on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+
+By default a pre-generated Access Key and Secret will be used. If you'd like to set your own password change the AccessKey and SecretKeyt in the values.yaml.
```
Comment (viglesiasce) [inline stable/minio/README.md]: Typo here `SecretKeyt`

File: stable/minio/templates/minio_secret.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: minio-user
```
Comment (viglesiasce) [inline stable/minio/templates/minio_secret.yaml]: This name and the name declared in the deployment (line 34) do not line up. This causes the pod to fail to start.

**Review by viglesiasce — CHANGES_REQUESTED** [review]
Unable to start this app due to mismatched secret names (minio-user vs $release-minio-user.

File: stable/minio/README.md (hunk 1 above)
Comment (hunter) [reply stable/minio/README.md]: Fixed

File: stable/minio/templates/minio_secret.yaml (hunk 2 above)
Comment (hunter) [reply stable/minio/templates/minio_secret.yaml]: Apologies, now fixed. Serves me right for only doing lint and dry-run testing.

**Review by viglesiasce — APPROVED** [review]

**Comment by viglesiasce** [comment]
Thanks @hunter!! Worked like a charm this time around.

**Pull request merged**